backends-velox/src/main/scala/org/apache/gluten/extension/CollectRewriteRule.scala

```diff
@@ -48,6 +51,25 @@ case class CollectRewriteRule(spark: SparkSession) extends Rule[LogicalPlan] {
     out
   }
 
+  private def needReplace(plan: LogicalPlan): Boolean = plan match {
+    // Do not replace collect in objectHashAggregate if disable objectHashAgg collectRewrite. When
```

Comment:
The comment is more appropriate to be the comment of the method.

Reply:
Updated, thanks.